.circleci/config.yml

```diff
@@ -324,10 +338,18 @@ workflows:
           requires:
             - 'Unminified Build'
       - 'Nomodule Tests':
+          name: 'Nomodule Tests (using << matrix.config >>-config.json)'
```

Comment:
Can you shorten the name to just `'Nomodule Tests (<< matrix.config >>)'`? They currently spill out of the box.

![image](https://user-images.githubusercontent.com/26553114/109050638-9c5eb300-76a7-11eb-8589-20cad22691f7.png)

Reply:
Done